src/site/pages/digital/src/containers/ICViewer/index.tsx

```diff
@@ -66,7 +73,13 @@ export const ICViewer = (() => {
             // Add input listener
             info.input.addListener((event) => {
                 const change = toolManager.onEvent(event, info);
-                if (change) renderer.render();
+                if (change) {
+                    renderer.render();
+                    const worldMousePos = camera.getWorldPos(info.input.getMousePos());
+                    const obj = FindPressableObjects(worldMousePos, info);
+                    if (isPressable(obj) && event.type === "click")
```

Comment:
Left this as `"click"` only since `"mousedown"` and `"mouseup"` would cause `pauseUpdates` to trigger if you just dragged your mouse across an object. I think it's alright if the callback doesn't pause right away if you press mouse down on a `Pressable` since it wouldn't actually toggle until you also released (or vice versa, basically just need an actual click event to toggle value and have reason for pausing).

Reply:
Hmmm let's talk about this in class, because how does this work with a Button?